workflow/src/ds-multisample-h5ad.lib.tengo

```diff
@@ -0,0 +1,73 @@
+ll := import("@platforma-sdk/workflow-tengo:ll")
+pt := import("@platforma-sdk/workflow-tengo:pt")
+render := import("@platforma-sdk/workflow-tengo:render")
+assets := import("@platforma-sdk/workflow-tengo:assets")
+pColumn := import("@platforma-sdk/workflow-tengo:pframes.pcolumn")
+parseH5adTpl := assets.importTemplate(":parse-multisample-h5ad")
+parseCsvTpl := assets.importTemplate(":parser-csv")
+
+util := import(":util")
+
+export {
```

Comment:
Same as above, please move functions outside the export block, to keep the same style.